core/src/main/resources/properties/item_dao.xml

```diff
@@ -112,7 +112,7 @@
 	</query>
 	<query>
 		<name>findAllItemsByVersionId</name>
-		<sql>select i.* from versioning_map v, item i 
+		<sql>select distinct i.* from versioning_map v, item i 
```

Comment:
Do we need to add another table in the query? I am trying to understand why "distinct" is used.